### PR TITLE
Updates. MonadResource in `servant-server` cleans a bit here

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: src/Servant/Multipart.hs [hunk 1]
```diff
@@ -244,49 +244,32 @@ instance ( FromMultipart a
...
+      fuzzyMultipartCTCheck (contentTypeH request)
+
+    bodyCheck () = do
+      mpd <- check opts :: DelayedIO MultipartData
+      case fromMultipart mpd of
+        Nothing -> liftRouteResult $ FailFatal
+          err400 { errBody = "fromMultipart returned Nothing" }
+        Just x -> return x
+
+    contentTypeH req = fromMaybe "application/octed-stream" $
```
Comment (alpmestan) [inline src/Servant/Multipart.hs]: Hmm, that should be `octet-stream` (the typo was there already though). Could you nonetheless quickly push a patch that fixes it as part of this PR? While we're at it.

**Comment by phadej** [comment]
@alpmestan file removal is registered inside [`tempFileBackEndOpts`](https://www.stackage.org/haddock/lts-7.16/wai-extra-3.0.19/src/Network.Wai.Parse.html#tempFileBackEndOpts)

**Comment by phadej** [comment]
Cannot enable travis for this repository, there's results from my: https://travis-ci.org/phadej/servant-multipart/builds

**Comment by alpmestan** [comment]
I've just enabled it. Merging.

**Pull request merged**